templates/staff/report_list.html

```diff
@@ -0,0 +1,142 @@
+{% extends "staff/base.html" %}
+
+{% load humanize %}
+{% load querystring_tools %}
+{% load selected_filter %}
+
+{% block metatitle %}Reports: Staff Area | OpenSAFELY Jobs{% endblock metatitle %}
+
+{% block breadcrumbs %}
+<nav class="breadcrumb-container breadcrumb--danger" aria-label="breadcrumb">
+  <div class="container">
+    <ol class="breadcrumb rounded-0 mb-0 px-0">
+      <li class="breadcrumb-item">
+        <a href="{% url 'staff:index' %}">Staff area</a>
+      </li>
+      <li class="breadcrumb-item active" aria-current="page">
+        Reports
+      </li>
+    </ol>
+  </div>
+</nav>
+{% endblock breadcrumbs %}
+
+{% block jumbotron %}
+<div class="jumbotron jumbotron-fluid jumbotron--danger pt-md-2">
+  <div class="container">
+    <h1 class="display-4">Reports</h1>
+  </div>
+</div>
+{% endblock jumbotron %}
+
+{% block staff_content %}
+<div class="container">
+  <div class="row">
+    <div class="col col-lg-3 col-xl-4">
+      <h2 class="h3">Filters</h2>
+
+      {% if request.GET %}
+      <div class="mb-3">
+        <a href="{% url 'staff:application-list' %}">Clear All</a>
+      </div>
+      {% endif %}
+
+      {% if orgs %}
+      <h3 class="h4">Orgs</h3>
+      <div class="btn-group-vertical w-100 mb-4" role="group" aria-label="Filter by org">
+        {% for org in orgs %}
+        {% is_filter_selected key="org" value=org.slug as is_active %}
+        <a
+          {% if is_active %}aria-pressed="true"{% endif %}
+          class="btn btn-outline-primary btn-block text-left {% if is_active %}active{% endif %}"
+          href="
+            {% if is_active %}
+              {% url_without_querystring org=org.slug %}
+            {% else %}
+              {% url_with_querystring org=org.slug %}
+            {% endif %}
+          "
+        >
+            {{ org.name }}
+        </a>
+        {% endfor %}
+      </div>
+      {% endif %}
+
+      {% if projects %}
+      <h3 class="h4">Projects</h3>
+      <div class="btn-group-vertical w-100 mb-4" role="group" aria-label="Filter by project">
+        {% for project in projects %}
+        {% is_filter_selected key="project" value=project.value as is_active %}
+        <a
+          {% if is_active %}aria-pressed="true"{% endif %}
+          class="btn btn-outline-primary btn-block text-left {% if is_active %}active{% endif %}"
+          href="
+            {% if is_active %}
+              {% url_without_querystring project=project.value %}
+            {% else %}
+              {% url_with_querystring project=project.value %}
+            {% endif %}
+          "
+        >
+            {{ project.title }}
+        </a>
+        {% endfor %}
+      </div>
+      {% endif %}
+
+      {% if users %}
+      <h3 class="h4">Authors</h3>
```

Comment:
Would "creators" rather than "authors" be more accurate on the staff pages? I'm thinking that more accurately covers both Interactive and Reports reports.